api/HTMLMeterElement.json

```diff
@@ -26,7 +26,10 @@
             "version_added": "11"
           },
           "safari": {
-            "version_added": "5"
+            "version_added": "6"
```

Comment:
Can you test if `<meter>` itself was also added in Safari 6 and if that entry needs updating? (Maybe split into a separate PR if it's complicated.)